tests/test_calibType.py

```diff
@@ -39,6 +47,8 @@ def setUp(self):
                                 {'exposure': 1235, 'detector': 0, 'filter': 'G'},
                                 {'exposure': 1234, 'detector': 1, 'filter': 'G'},
                                 {'exposure': 1235, 'detector': 1, 'filter': 'G'}])
+        self.exposure1 = FakeExposure(SEQNAME='sequencer A', SEQFILE='my.seq', SEQCKSUM='abcdef')
+        self.exposure2 = FakeExposure(SEQNAME='sequencer B', SEQFILE='my.seq')
```

Comment:
This seems good to add, but I can't quite work out where this is being exercised. Shouldn't there be something with a `self. assertRaises()` and a config which says to raise, and ideally one with the warnings too?

Reply:
This doesn't test that due to the same issue with calibs not having an `IsrTaskConfig` associated with them.

Reply:
Ah, OK, thanks.